docs/_interface/mining.md

```diff
@@ -1,191 +1,204 @@
 
-### Installing ethminer
 
-To get ethminer, you need to install the ethminer binary package or build it from source.
-See <https://github.com/ethereum-mining/ethminer/#build> for the official ethminer
-build/install instructions. At the time of writing, ethminer only provides a binary for
-Microsoft Windows.
+## CPU vs GPU
 
-### Using ethminer with geth
+Participating in Ethereum's PoW mining requires running an algorithm called 
+["Ethash"](https://ethereum.org/en/developers/docs/consensus-mechanisms/pow/mining-algorithms/ethash). Geth includes
+a CPU miner which runs Ethash within the Geth process. This might be useful for mining on some testnets. However, this is CPU mining is not viable on Ethereum Mainnet because CPU miners are easily out-competed by more efficient GPU miners. GPU mining is the recommended method for mining real ether on Ethereum Mainnet, but it is not part of the standard Geth installation. To mine using GPUs an additional piece of third-paty software is required. The recommended GPU mining software is [Ethminer](https://github.com/ethereum-mining/ethminer).
 
-First create an account to hold your block rewards.
+Regardless of the mining method, the blockchain must be fully synced before mining is started, otherwise the miner will build on an incorrect chain, invalidating the block rewards. 
 
-    geth account new
 
-Follow the prompts and enter a good password. **DO NOT FORGET YOUR PASSWORD**. Also take
-note of the public Ethereum address which is printed at the end of the account creation
-process. In the following examples, we will use 0xC95767AC46EA2A9162F0734651d6cF17e5BfcF10
-as the example address.
+## GPU Mining
 
-Now start geth and wait for it to sync the blockchain. This will take quite a while.
+### Installing Ethminer
 
-    geth --http --miner.etherbase 0xC95767AC46EA2A9162F0734651d6cF17e5BfcF10
+The Ethminer software can be installed from a downloaded binary or built from source. Either way, the relevant downloads 
+and installation instructions are available from the [Ethminer Github](https://github.com/ethereum-mining/ethminer/#build). Standalone executables are available for Linux, macOS and Windows.
 
-To monitor the syncing, in another terminal you can `attach` the geth JavaScript console to the running node like so:
+### Using Ethminer with Geth
 
-    geth attach http://127.0.0.1:8545
+An account to receive block rewards must first be created or imported into Geth. To create a new account:
```

Comment:
We don't need to create the account with geth (or import it with geth). The address of the account is sufficient to start mining to it